### ISSUE TITLE
ScrollContainer doesn't trigger onClick event

### ISSUE THREAD

**Issue by sudhir-pradhan** [Norserium/react-indiana-drag-scroll#25]
Popovers inside ScrollContainer,  don't get dismissed when users click off them.
If users click inside the scroll component to open another popover or try to scroll the div, the active/previous popover does not disappear. It seems like `onClick` event doesn't get fired or something. How can I achieve this? [demo code ](https://stackblitz.com/edit/so-60079855-dismiss-popover-react-indiana-drag-scroll)
```
 handleOnClick = (e) => {
    console.log("clicked"); // no output to console
  };
<ScrollContainer
          className="scroll-container attendance-scroller test"
          hideScrollbars={false}
          nativeMobileScroll={true}
          onClick={this.handleOnClick}
        >
      ...
        </ScrollContainer>
```

**Comment by Norserium** [comment]
Hello, @sudhir-pradhan!

I confirm that problem exists. The `stopPropagation` method is called in `onMouseDown` and `onMouseUp` methods, I suppose that is the reason of your bug.

Me need time to investigate if the disabling of this call is possible. Probably, I should add `stopPropagation` prop, that will control this behavior.

**Comment by sudhir-pradhan** [comment]
I was just answering a question on SO https://stackoverflow.com/questions/60079855/react-indiana-drag-scroll-component-interfering-with-ant-design-popover/60151447#60151447.
I had to attach `onClick` event to the `table` inside `ScrollContainer`.

**Comment by Norserium** [comment]
This bug was fixed in `1.6.0`. I've added the answer with this information to the corresponding question.
